### PR TITLE
Update to support ckgedit and ckgdoku

### PR DESCRIPTION
**Pull request by turnermm**
I had a user of ckgdoku who wants to use this plugin but differences in the id of the save button prevented this.   I wasn't able to accomodate him in ckgdoku, so made these updates.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -24,7 +24,18 @@ jQuery(function() {
...
-    var $savebutton = jQuery("#edbtn__save");
+    var ckgd =  jQuery( "input[type=checkbox][name=ckgdoku]:checked" ).val();
+    var ckge =  jQuery( "input[type=checkbox][name=ckgedit]:checked" ).val();
+    var m_class;
+    if(typeof ckgd == 'string' || typeof ckge == 'string') {       
+        var $savebutton = jQuery("#save_button");
+        m_class = "plugin_enforcesummary_missing";
+    }
+    else {
+        $savebutton = jQuery("#edbtn__save");
```
Comment (michitux) [inline script.js]: This declares a global variable as far as I'm aware. I think `$savebutton` should be declared before the `if`-statement.

File: script.js [hunk 2]
```diff
@@ -24,7 +24,18 @@ jQuery(function() {
...
     var $minoredit = jQuery('#minoredit');
-    var $savebutton = jQuery("#edbtn__save");
+    var ckgd =  jQuery( "input[type=checkbox][name=ckgdoku]:checked" ).val();
+    var ckge =  jQuery( "input[type=checkbox][name=ckgedit]:checked" ).val();
+    var m_class;
+    if(typeof ckgd == 'string' || typeof ckge == 'string') {       
+        var $savebutton = jQuery("#save_button");
+        m_class = "plugin_enforcesummary_missing";
+    }
+    else {
```
Comment (michitux) [inline script.js]: This linebreak is strange, the `else` shouldn't be on a new line.

**Comment by turnermm** [comment]
Thanks.

**Comment by turnermm** [comment]
I was curious about the vulnerability of the `$savebutton`  variable.  I have always assumed that kind of construction is safe, but your query put me in some doubt.  So, I did some checking.  It turns out that in Javascript you can declare a variable anywhere in a function and it is then local to that function.  It's called `variable` hoisting.  

.

**Comment by michitux** [comment]
Okay, yes, you are right - and I've just learned something new about JavaScript. I wrongly assumed the variable was only declared if the `if`-branch was executed. However, as you correctly pointed out this is not true, the variable is always declared because it is declared somewhere in that function which is really strange behavior if you know other languages. I still think it was good to change this to make it obvious/easier to read.

**Comment by turnermm** [comment]
I agree.  I cam from a C background.  

Are you planning to merge my pull or did you want me to keep a local copy for use with ckgedit?

**Comment by michitux** [comment]
This is not my plugin (even though for some reason @lupo49 gave me push access), that's why I did not directly merge it. As there has been no reaction from him I'll just merge this now.

**Comment by turnermm** [comment]
Thanks.

**Pull request merged**